tests/authors.test.js

```diff
@@ -0,0 +1,137 @@
+const { expect } = require('chai');
+const { Author } = require('../src/models');
+const request = require('supertest');
+const app = require('../src/app');
+
+describe('/authors', () => {
+   before(async () => Author.sequelize.sync());
+
+   beforeEach(async () => {
+      await Author.destroy({ where: {} })
+   })
+
+   describe('with no records in the database', () => {
+        describe('POST/authors', () => {
+            it('creates a new author in the database', async () => {
+               const response = await request(app).post('/authors').send({ 
+
+                  author: 'Dan Hembery'
+
```

Comment:
Really don't need all that extra space here. Same applies later on in the test
```suggestion
                  author: 'Dan Hembery'
```